ansible/roles/gen_kubernetes_config/templates/forkmon.yaml.j2

```diff
@@ -0,0 +1,64 @@
+# ANSIBLE GENERATED FILE - DO NOT EDIT MANUALLY
+forkmon:
+  fullnameOverride: forkmon
+  image:
+    repository: skylenet/nodemonitor
+    tag: darkmode@sha256:84cb51d00aea5e226e7e32ccef298b03c1853afc42f5cbd291c4f5bffe3a6a7b
+  ingress:
+    enabled: true
+
+    className: ingress-nginx-public
+
+    hosts:
+      - host: forkmon.{{ ethereum_network_name }}.ethpandaops.io
+        paths:
+          - path: /
+            pathType: Prefix
+  network: {{ ethereum_network_name }}
+  endpoints:
+  # Rpc
+  - addr: http://nethermind-lodestar-001:8545
+    name: rpc-nethermind-lodestar
+{% for host in (groups['all']) | sort %}
+  - addr: https://<path:/secrets/services/services.enc.yaml#ethereum | jsonPath {.testnets.4844-devnets.node_ingress.combined}>@rpc.{{ hostvars[host]['inventory_hostname'] }}.srv.{{ ethereum_network_name }}.ethpandaops.io
+    name: {{ hostvars[host]['inventory_hostname'] }}
+{% endfor %}
+{% raw %}
+  config: |
+    # How often to reload data from the nodes
+    reload_interval = "10s"
+    # If specified, a http server will serve static content here
+    server_address = "0.0.0.0:{{ .Values.httpPort}}"
+    # Shown in the document title, if specified
+    chain_name="{{ .Values.network }}"
+    # Third party providers
+    infura_key          = "<INFURA_API_KEY>"
```

Comment:
wait this part doesn't need to be here no?

Reply:
thats just standard example. Not being used.